.gitlab-ci.yml

```diff
@@ -245,6 +245,15 @@ build:base+async:
     COQ_EXTRA_CONF: "-native-compiler yes -coqide opt"
     COQUSERFLAGS: "-async-proofs on"
 
+build:quick:
+  <<: *build-template
+  stage: test
```

Comment:
Putting this in the test stage makes it impossible to use it in the validate job.

Reply:
ok thanks I'll fix this.